media_driver/linux/common/os/mos_os_specific.c

```diff
@@ -5780,53 +5780,6 @@ MOS_VDBOX_NODE_IND Mos_Specific_GetVdboxNodeId(
         return idx;
     }
 
-#if defined(MEDIA_EXT) && !defined(ANDROID)
```

Comment:
There is other code for VDBOX balancing under MEDIA_EXT - I suggest to remove it.